src/plugins/data/public/antlr/opensearch_sql/code_completion.ts

```diff
@@ -86,7 +85,7 @@ export const getSuggestions = async ({
           tableNames,
           suggestions.suggestValuesForColumn as string,
           api,
-          connectionService
+          services
```

Comment:
might need to remove these lines from here, since the other autocomplete prs were removed from this branch. this change was added in this commit, would only need to revert the changes done on this file: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/7492/commits/bf057f2ac8468aba653b6ad2ea675413da1f1178#diff-f67c8b1b68c9ae8eeb8ea7d7dea38e59d32b7738dec7c9db5763cf74b96aa92fR46